acra/acra-in-depth/data-structures/acrastruct.md

```diff
@@ -50,25 +50,25 @@ To generate AcraStructs, AcraServer/AcraTranslator/AcraWriter uses a public key
 
 The generation contains the following steps:
 
-- Generates a keypair of throwaway keys using Themis EC key generator:<br/>
+- Generate a keypair of throwaway keys using Themis EC key generator:<br/>
```

Comment:
i think it makes sense, because previously it said "AcraServer does the following steps: ... - Generates..".

Now we say "The generation contains the following steps:", so it makes sense not to use `-s` anymore

Reply:
The phrase starts from: "The generation contains the following steps:" and "s" seems not appropriate - we have to either use verb as a verb: step1 - to generate..., step2 - to encrypt. Or rephrase initial sentence like: AcraStruct can be generated with a specific tool that: generates, encrypts, etc.